### PR TITLE
Guarin lig 1010 multiple object level run docs

### PR DESCRIPTION
**Pull request by guarin**
* Update docker configuration
* Add information about crop dataset naming to object level docs

### CODE REVIEW

File: docs/source/docker/advanced/object_level.rst [hunk 1]
```diff
@@ -119,9 +119,9 @@ For more information regarding the predictions format please see :ref:`ref-docke
 Selection on Object Level
 -------------------------
 Once you have everything set up as described above, you can run selection on
-object level by setting the `object_level.task_name` argument in the Lightly
-docker config. The argument should be set to the task name you used for your
-predictions. If you uploaded the predictions to `.lightly/predictions/vehicles_object_detections`
+object level by setting the `object_level.task_name` argument in the :ref:`docker configuration <ref-docker-configuration>`. 
+The argument should be set to the task name you used for your predictions. 
+If you uploaded the predictions to `.lightly/predictions/vehicles_object_detections`
```
Comment (philippmwirth) [inline docs/source/docker/advanced/object_level.rst:124]: You could add an `E.g.` here

**Review by philippmwirth — CHANGES_REQUESTED** [review]
Somehow this doesn't contain any concrete information about how to run the workflow twice, does it?

File: docs/source/docker/advanced/object_level.rst (hunk 1 above)
Comment (guarin) [reply docs/source/docker/advanced/object_level.rst]: Thanks, added it :)

**Comment by guarin** [comment]
> Somehow this doesn't contain any concrete information about how to run the workflow twice, does it?

Yes, I did not add it first because by default it should just work. But I added a small section that explains it, hope it is understandable :)

**Review by philippmwirth — APPROVED** [review]
looks good!

**Pull request merged**